### PR TITLE
sanitize URL

### PR DESCRIPTION
**Pull request by lpranam**
* Target version: distro/collabora/co-22-05 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/js/sanitize-url.js [hunk 1]
```diff
@@ -1,43 +1,46 @@
...
   'use strict';
 
-  var invalidPrototcolRegex = /^(%20|\s)*(javascript|data)/im;
-  var ctrlCharactersRegex = /[^\x20-\x7EÀ-ž]/gmi;
-  var urlSchemeRegex = /^([^:]+):/gm;
-  var relativeFirstCharacters = ['.', '/'];
-  
+  var invalidProtocolRegex = /^([^\w]*)(javascript|data|vbscript)/im;
+  var htmlEntitiesRegex = /&#(\w+)(^\w|;)?/g;
+  var htmlCtrlEntityRegex = /&(newline|tab);/gi;
```
Comment (eszkadev) [inline browser/js/sanitize-url.js:6]: does it work? shouldn't be `\n` and `\t` ? I think above means that these characters are matching: "n", "e", "w", "l", "i", "n", "e"

**Comment by eszkadev** [comment]
these sanitize-url changes are update of the library to the new version? or done by you?

**Comment by lpranam** [comment]
they are done by me, i did not update entire library but just our code.

File: browser/js/sanitize-url.js (hunk 1 above)
Comment (eszkadev) [reply browser/js/sanitize-url.js:6]: this is backport from source library

**Review by eszkadev — APPROVED** [review]

**Pull request merged**